markdown/report.Rmd

```diff
@@ -0,0 +1,416 @@
+The trend over time by deprivation category is presented in Figure A1.3 (see [List of Tables](LINK) or [Appendix A1](#AppendixA1)). All deprivation categories show a similar trend over time, with a slight increase in the percentage of last six months spent at home or in the community for individuals in the last two or three years. For each year in the trend (2010/11 to 2018/19) there is little variation across the different deprivation categories.
+
+### Urban Rural Classification
+The Scottish Government 6-fold [urban rural classification](http://www.gov.scot/Topics/Statistics/About/Methodology/UrbanRuralClassification) has been used to present information on whether people live in Urban or Rural areas and if there is any difference in the amount of time spent at home or in a community setting in the six months prior to death (Figure 5).
+
+#### UR Chart HERE
+
+1.	Figures for 2018/19 are incomplete and will be revised in October 2019 publication. See [Appendix 2](#Appendix2) for more information.
+
+The percentage of the last six months of life spent at home or in the community ranged from 89.0% for residents in large urban areas to 90.3% for residents from remote rural areas (Figure 5; Table 5 in [data tables](LINK)). Individuals living in remote rural areas spent, on average, approximately two extra days at home or in a community setting in the last six months of life compared to those living in large urban areas.
+
+Trend data on the percentage of the last six months of life spent at home or in a community setting by Urban/Rural classification are presented in Figure A1.4 (see [List of Tables](LINK) or [Appendix A1](LINK)).
+
+
+
+####### Glossary
+```{r kable_glossary, echo = FALSE}
+    knitr::kable(glossary, col.names = NULL, align = c('l','l'))
+```
+
+####### List of Tables
+<br>
+```{r kable_ListOfTables, echo = FALSE}
+knitr::kable(ListofTables, col.names = c("**File name**", "**File and size**"), align = c('l','l'))
+```
+
+<Div custom-style = "Heading 2_pgbrk">Contact</div>
+**Federico Centoni, Information Analyst**  
+Health and Social Care  
+0131 314 1532  
+federico.centoni@nhs.net  
+
+**Alice Byers, Senior Information Analyst**  
+Health and Social Care  
+0131 314 1725  
+alice.byers@nhs.net
+
+**Aghimien Iyayi-Igbinovia, Principal Information Analyst**  
+Health and Social Care  
+0141 207 1616  
+aghimien.iyayi-igbinovia@nhs.net  
+
+<br>
+
+## Further Information
+Further Information can be found on the [ISD website](http://www.isdscotland.org/).  
+
+For more information on Palliative and End of Life Care see the [End of Life Care section of our website](http://www.isdscotland.org/Health-Topics/Health-and-Social-Community-Care/End-of-Life-Care/).
+
+The next release of this publication will be October 2019.  
+
+<br>
+
+## Rate this publication
```

Comment:
Change it to `<Div custom-style = "Heading 2_nopgbrk">Rate this publication</div>` after using the updated "NATIONAL_STATS_REPORT_TEMPLATE.docx"